core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -91,14 +105,28 @@ object OpenApiAccess {
         return path.toFile().readText()
     }
 
-    private fun connectToServer(openApiUrl: String, attempts: Int): Response {
+    private fun connectToServer(openApiUrl: String, authentication : HttpWsAuthenticationInfo, attempts: Int): Response {
 
         for (i in 0 until attempts) {
             try {
-                return ClientBuilder.newClient()
-                        .target(openApiUrl)
-                        .request(MediaType.APPLICATION_JSON_TYPE)
-                        .get()
+
+                var currentReq = ClientBuilder.newClient()
+                                              .target(openApiUrl)
+                                              .request(MediaType.APPLICATION_JSON_TYPE)
+
+                for(head in authentication.headers) {
```

Comment:
this is incomplete. these headers can be empty, and auth done with `endpointCallLogin`. i pushed in `master` some more clarifications for `HttpWsAuthenticationInfo`

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.